codalab/rest/bundles.py

```diff
@@ -277,20 +282,21 @@ def _create_bundles():
         # Save bundle into model
         local.model.save_bundle(bundle)
 
-        # Inherit worksheet permissions
-        group_permissions = local.model.get_group_worksheet_permissions(
-            request.user.user_id, worksheet_uuid
-        )
-        set_bundle_permissions(
-            [
-                {
-                    'object_uuid': bundle_uuid,
-                    'group_uuid': p['group_uuid'],
-                    'permission': p['permission'],
-                }
-                for p in group_permissions
-            ]
-        )
+        if not detached:
+            # Inherit worksheet permissions
```

Comment:
Interestingly no bug was thrown for not having this check before. Maybe add a comment to say that if it's detached, then only the user will have all permissions on the bundle.

Reply:
thanks for the advice. appiled!